generate_reports.py

```diff
@@ -0,0 +1,316 @@
+import numpy as np
+
+from common import get_column_data_from_alias
+from common import get_month_string_from_number
+from load_data_from_file import load_month_data, load_year_data
+from WeatherData import WeatherData
+
+
+TEXT_STYLE_NORMAL = 0
+TEXT_STYLE_BOLD = 1
+TEXT_STYLE_LIGHT = 2
+TEXT_STYLE_ITALICIZED = 3
+TEXT_STYLE_UNDERLINED = 4
+TEXT_STYLE_BLINK = 5
+
+TEXT_COLOR_BLACK = 30
+TEXT_COLOR_RED = 31
+TEXT_COLOR_GREEN = 32
+TEXT_COLOR_YELLOW = 33
+TEXT_COLOR_BLUE = 34
+TEXT_COLOR_PURPLE = 35
+TEXT_COLOR_CYAN = 36
+TEXT_COLOR_WHITE = 37
+
+BACKGROUND_COLOR_BLACK = 40
+BACKGROUND_COLOR_RED = 41
+BACKGROUND_COLOR_GREEN = 42
+BACKGROUND_COLOR_YELLOW = 43
+BACKGROUND_COLOR_BLUE = 44
+BACKGROUND_COLOR_PURPLE = 45
+BACKGROUND_COLOR_CYAN = 46
+BACKGROUND_COLOR_WHITE = 47
+
+
+def change_text_color(style=0, color=37, background_color=40):
```

Comment:
improve function name, not clear what the function is doing from reading the name!